tests/rptest/tests/random_node_operations_test.py

```diff
@@ -122,8 +118,17 @@ def _setup_test_scale(self, num_to_upgrade):
             f"running test with: [message_size {self.msg_size},  total_bytes: {self.total_data}, message_count: {self.msg_count}, rate_limit: {self.rate_limit}, cluster_operations: {self.node_operations}]"
         )
 
-    def _start_redpanda(self, number_nodes_to_upgrade,
-                        enable_controller_snapshots):
+    def _start_redpanda(self, number_nodes_to_upgrade, with_tiered_storage):
+
+        if with_tiered_storage:
+            si_settings = SISettings(self.test_context,
+                                     cloud_storage_enable_remote_read=True,
+                                     cloud_storage_enable_remote_write=True)
```

Comment:
I don't know what the usual runtime of this test is, but you can use `fast_uploads=True` to make redpanda upload more eagerly.